package.json

```diff
@@ -15,6 +15,10 @@
     "prettier-base": "prettier . --ignore-path .gitignore --ignore-unknown",
     "format": "yarn prettier-base --write",
     "format:check": "yarn prettier-base --check",
+    "cypress": "cypress open",
+    "cypress:headless": "cypress run",
+    "e2e": "start-server-and-test 'yarn dev' http://localhost:3000 'yarn cypress'",
+    "e2e:headless": "start-server-and-test 'yarn serve' http://localhost:3000 'yarn cypress:headless'",
```

Comment:
Handy commands for running Cypress locally.

Reply:
Cool!